### DISCUSSION TITLE
The Model of scaling  across species

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1715] [Models]
Hi everyone 
       Recently, I leran the model of scaling  across species followed esQlabs tutorials, I still wonder how the scaling realized in the model. It  seems to be taking CL from animals and putting it directly into humans. Would anyone explain the mechanisam for me? I will fell every appreciated!
Thankyou!

**Comment by zbq-2019** [comment]
Search the term allometric scale  in Google.
Energy consumption is of extreme importance in understanding interspecies variation in pharmacokinetics, because it influences in one or
another way the amount of energy to be directed towards elimination of foreign chemicals including drugs. An increase in body mass is generally associated with a progressive reduction in the metabolic rate per unit mass, in turn influencing the turnover rate at the cellular level

**Comment by SCZ1174** [comment]
Personal opinion. In the tutorial, species scaling is implemented using "specific clearance". It is a "well-stirred model" (unsure) where you can derive a "specific clearance" based on realistic parameters, and then the system scales according to the individuals or populations you build. The only parameter used in the simulation is the CL_spec that is independent of the original individual. For example you can scale between ages, between species. 
I don't know if I'm making sense, so I hope this helps.
Also, recommend watching the previous review #1442 #818
